src/libraries/Microsoft.Internal.Runtime.WindowsDesktop.Transport/src/Microsoft.Internal.Runtime.WindowsDesktop.Transport.proj

```diff
@@ -13,6 +13,12 @@
     <NoWarn>$(NoWarn);NU5131</NoWarn>
   </PropertyGroup>
 
+  <!-- Always generate this package during servicing to flow the dependency to WindowsDesktop. -->
+  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
```

Comment:
Because the 6.0 change was directly committed to release/6.0 when release/6.0 meant 6.0.1. That branch will never again mean anything else than servicing, that's why the condition wasn't needed. This change is more universal and works in both servicing, RTM and main.